projects/angular/src/button/button-group/button-group.ts

```diff
@@ -111,6 +128,22 @@ export class ClrButtonGroup implements AfterContentInit {
     this.menuButtons = tempInMenuButtons;
   }
 
+  handleFocusOnMenuOpen() {
+    this.toggleService.openChange
+      .pipe(
+        first(event => event === true),
+        delay(0),
+        map(event => {
```

Comment:
I would rename these `event` parameters to `open` to make the code more understandable.